tests/projects/test_steps.py

```diff
@@ -161,6 +161,7 @@ def test_build_docker_image_raises_with_auto_and_existing_dockerfile():
         Path("Dockerfile").unlink()
 
 
+@pytest.mark.flaky(max_runs=3)
```

Comment:
Isn't that the default max runs?

It seems like very bad form to mark a bunch of tests as flaky like this — what's going on here?

Reply:
The tests that run with a real Docker client fail intermittently in CI with this error when calling `docker.from_env()`:
```
TypeError: object.__new__() takes exactly one argument (the type to instantiate)
```
I have not been able to determine what's causing this error, but it generally resolves itself on a re-run. Have you seen this error or one like it before?

Reply:
Hm no I haven't that's really weird. I guess let's track it in an issue 🤷‍♂️ don't want to hold ya up.